src/Model/FieldType/URLField.test.ts

```diff
@@ -45,8 +45,8 @@ describe("FieldType: URLField", () => {
     expect(URLField.serialize).to.be.true;
   });
```

Comment:
There is no transform test here.

Reply:
Never mind, I forget that the tests are preceding the actual code.  Read on below!